frontend/src/components/ResourceNameTooltip.tsx

```diff
@@ -9,52 +9,87 @@ import {
   StackItem,
   Tooltip,
 } from '@patternfly/react-core';
-import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import { CopyIcon, InfoAltIcon, OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
 import { K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
 import '~/pages/notebookController/NotebookController.scss';
+import { EitherNotBoth } from '~/typeHelpers';
 
-type ResourceNameTooltipProps = {
-  resource: K8sResourceCommon;
-  children: React.ReactNode;
+type ResourceNameTooltipProps = EitherNotBoth<
+  { resource: K8sResourceCommon },
+  { resourceName: string; resourceKind: string }
+> & {
+  children?: React.ReactNode;
 };
```

Comment:
Not sure we want to change the API 🤔 I wonder if you can just craft a "fake" object... we just need a metadata object really